### ISSUE TITLE
Proposal to shorten waiting time for automatic approval of leadership meeting minutes

### ISSUE THREAD

**Issue by jonathanwheeler01** [carpentries/trainers#294]
## Summary

Minutes of trainers leadership committee meetings are posted to this repository via a pull request (PR). Once approved, the PR is merged into the main repository branch. Approval from leadership committee members can be done through the PR, or via Slack. In the absence of approval from the full committee, minutes can be considered approved and the PR merged after a month from the request date if there are no corresponding discussions or questions. The purpose of this proposal is to shorten the wait time for merging the PR from a month to two weeks.

## Background

The trainers leadership committee meets once each month, with a second optional monthly meeting scheduled as needed to address committee business. Minutes are recorded at each meeting on a non-public system, and a standing action item is to review the minutes for redactions. Any redactions are either completed during the meeting or soon after the meeting during conversion of the minutes to Markdown. Because of this real time vetting process, minutes can generally be made public after they are converted to Markdown.

Once the minutes are in Markdown format, they are submitted to this repository as a pull request (PR). Though PRs are publicly accessible, the minutes are not officially merged into the main repository branch until approved by the trainers leadership committee (TLC). As noted above, approval can be provided via Slack or within the PR. Once the minutes are approved, the PR is merged into the main branch of the repository. Explicit approval is not required - the PR can also be merged after a month if there is no ongoing discussion or unresolved questions. 

The TLC is interested in determining if one month allows for a timely approval of the meeting minutes, or if a shorter wait time of two weeks is appropriate.

## Implementation

The process of publishing the minutes will remain the same as described above. The only difference is that members of the TLC will be authorized to merge corresponding pull requests two weeks after the request is posted.

### Pros

- This can be implemented with no additional process changes.
- Minutes become most accessible via the main repository branch sooner, in general ahead of the next TLC meeting.
- Reduces the number of outstanding pull requests.

### Cons

- Taking schedules, time zones, professional and personal leave time, etc. into account, two weeks may not be enough time for a considered review of the minutes/pull request.
- Less time for TLC members who may have missed a meeting to review the minutes,
- Possibility of rushing approval at the expense of complete, accurate minutes. This can result in less transparency overall.

## Possible Alternatives

- Make no change and continue with the one-month waiting period for approval.

**Comment by ndporter** [comment]
I like this proposal (per Martha's rules)

**Comment by ragamouf** [comment]
I like this proposal (per Martha's rules)

**Comment by annajiat** [comment]
I support this proposal.

**Comment by ragamouf** [comment]
Approved in TLC meeting 14/15 Nov 2024
